tests/util/xcm.ts

```diff
@@ -411,6 +415,321 @@ export class XcmFragment {
+        query_id,
+        max_weight,
+      },
+    });
+    return this;
+  }
+
+  // Add a `ClearTransactStatus` instruction
+  clear_transact_status(): this {
+    this.instructions.push({
+      ClearTransactStatus: null as any,
+    });
+    return this;
+  }
+
+  // Add a `UniversalOrigin` instruction
+  universal_origin(junction: any): this {
+    this.instructions.push({
+      UniversalOrigin: junction,
+    });
+    return this;
+  }
+
+  // Add a `ExportMessage` instruction
+  export_message(
+    network: "Any" | XcmV3JunctionNetworkId["type"] = "Any",
+    destination: number,
+    xcm: any
+  ): this {
+    this.instructions.push({
+      ExportMessage: {
+        network,
+        destination: { X1: { Parachain: destination } },
+        xcm,
+      },
+    });
+    return this;
+  }
+
+  // Add a `LockAsset` instruction
+  lock_asset(
+    multilocation: any = this.config.assets[0].multilocation,
+    fungible: any = this.config.assets[0].fungible,
+    unlocker: any = this.config.assets[0].multilocation
+  ): this {
+    this.instructions.push({
+      LockAsset: {
+        asset: {
+          id: {
+            Concrete: multilocation,
+          },
+          fun: {
+            Fungible: fungible,
+          },
+        },
+        unlocker,
+      },
+    });
+    return this;
+  }
+
+  // Add a `UnlockAsset` instruction
+  unlock_asset(
+    multilocation: any = this.config.assets[0].multilocation,
+    fungible: any = this.config.assets[0].fungible,
+    target: any = this.config.assets[0].multilocation
+  ): this {
+    this.instructions.push({
+      UnlockAsset: {
+        asset: {
+          id: {
+            Concrete: multilocation,
+          },
+          fun: {
+            Fungible: fungible,
+          },
+        },
+        target,
+      },
+    });
+    return this;
+  }
+
+  // Add a `NoteUnlockable` instruction
+  note_unlockable(
+    multilocation: any = this.config.assets[0].multilocation,
+    fungible: any = this.config.assets[0].fungible,
+    owner: any = this.config.assets[0].multilocation
+  ): this {
+    this.instructions.push({
+      NoteUnlockable: {
+        asset: {
+          id: {
+            Concrete: multilocation,
+          },
+          fun: {
+            Fungible: fungible,
+          },
+        },
+        owner,
+      },
+    });
+    return this;
+  }
+
+  // Add a `RequestUnlock` instruction
+  request_unlock(
+    multilocation: any = this.config.assets[0].multilocation,
+    fungible: any = this.config.assets[0].fungible,
+    locker: any = this.config.assets[0].multilocation
+  ): this {
+    this.instructions.push({
+      RequestUnlock: {
+        asset: {
+          id: {
+            Concrete: multilocation,
+          },
+          fun: {
+            Fungible: fungible,
+          },
+        },
+        locker,
+      },
+    });
+    return this;
+  }
+
+  // Add a `SetFeesMode` instruction
+  set_fees_mode(jit_withdraw: boolean = true): this {
```

Comment:
I use snake case convention in this parameter for it to match directly with the actual `jit_withdraw` field of the instruction. If I used `jitWithdraw`, it wouldn't work because of encoding errors.